src/SFA.DAS.ApplyService.Web/Views/Application/Standard/FindStandard.cshtml

```diff
@@ -40,7 +40,7 @@
                         <span id="search-string-error" class="govuk-error-message">Enter 2 or more characters to search</span>
                     }
 
-                    <input class="govuk-input govuk-input--width-20 ui-autocomplete-input @(!ViewData.ModelState.IsValid || showErrors ? "govuk-input--error" : "")" id="standard-name" name="StandardToFind" asp-for="@Model.StandardToFind"type="text" value="" autocomplete="off">
+                    <input class="govuk-input govuk-input--width-20 ui-autocomplete-input @(!ViewData.ModelState.IsValid || showErrors ? "govuk-input--error" : "")" id="standard-name" name="StandardToFind" asp-for="@Model.StandardToFind"type="text" value="" autocomplete="off" maxlength="500">
```

Comment:
In a subsequent change a similar field is increase from 500 to 10000 - this is given a new 500 limit though - inconsistent?

Reply:
This is the same file, so is outdated.